network/network_test.go

```diff
@@ -9,12 +9,13 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"errors"
-	"math"
 	"net"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/ethereum/go-ethereum/common/math"
```

Comment:
If this one was imported just for `MaxUint64`, we should use the native `math` library instead.